Audiobook Player/PlayerManager.swift

```diff
@@ -250,6 +261,8 @@ extension PlayerManager: AVAudioPlayerDelegate {
             return
         }
         
+        defaults.set(self.identifier, forKey: UserDefaultsConstants.lastPlayedBook);
```

Comment:
(This is just a comment) This looks interesting, are you planning on loading the bottom bar with the last played book on the next launch? I really like that idea 🤔

Reply:
That's the plan! I just forgot to remove this line from the commit since it didn't play into the feature. There's a lot we can do if we have the last listened to book - I was also thinking of adding an opt-in setting to autoplay last book on launch.